hw/dv/dpi/usbdpi/usb_transfer.c

```diff
@@ -3,7 +3,9 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include <assert.h>
+#include <string.h>
 
+#include "usb_utils.h"
 #include "usbdpi.h"
 
 // Set up all of the available transfer descriptors
```

Comment:
Counting down from the highest index to the lowest naturally links each descriptor to the one above it such that they are initially linked in order, and it just feels more elegant to use a while loop than a for loop when downcounting. It can, of course, be done using any of the loop statements.